### PR TITLE
Test add delete 

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
- Use Babel to use ES6 features
- Test add and delete functionality with jest 
- Test add and delete functionality with jest in the DOM environment

### CODE REVIEW

File: src/__mocks__/handlers.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                </p>
+                <p>
+                    <i data-index="${index}" class="remove-item">&times</i>
+                    <span class="drag-around">&#8942</span>
+                </p>
+            </li>
+            `,
+    );
+
+    clearAll?.insertAdjacentHTML('beforebegin', listItems.join(''));
```
Comment (mikethreels) [inline src/__mocks__/handlers.js:49]: - [x] great job on writing the tests 👍 however not all tests are passing as you can see in the below print screen. which is due to this line of code.
Capture

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__mocks__/handlers.js (hunk 1 above)
Comment (Lucas-Masaba) [reply src/__mocks__/handlers.js:49]: Hello Michael Threels. Thank you for making good observations. In the code review, you commented that some tests failed. Although, when I and both my coding partners tested on our PCs, all the tests passed. How can we solve this problem?
image

**Comment by Lucas-Masaba** [comment]
All tests ran on me and my partners end and here is an image proving that all 4 tests are successful 
image

File: src/__mocks__/handlers.js (hunk 1 above)
Comment (helman101) [reply src/__mocks__/handlers.js:49]: For me is everything working, so you are approved 🥳 🎉 

imagen

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi guys 🥇, 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**